Data-Structures/graph/graph.js

```diff
@@ -31,22 +30,47 @@ class Graph {
   }
 
   getNodes() {
-    if (!this.adjacencyList) return this.adjacencyList;
+    if (!this.adjacencyList.length) return this.adjacencyList;
 
     return this.adjacencyList.map(i => i.head.value);
   }
 
   getNeighbors(node) {
-    if (!this.adjacencyList) return [];
+    if (!this.adjacencyList.length) return [];
 
     let nodeConnections = this.adjacencyList.find(i => i.head.value === node);
     if (nodeConnections) return nodeConnections.toString().split(', ').slice(1).map(i => JSON.parse(i));
```

Comment:
Not sure why you're using `toString().split()` here instead of just `nodeConnections.slice(1)`?